src/test/ui/mismatched_types/closure-arg-count.stderr

```diff
@@ -2,8 +2,10 @@ error[E0593]: closure is expected to take 2 arguments, but it takes 0 arguments
   --> $DIR/closure-arg-count.rs:15:15
    |
 LL |     [1, 2, 3].sort_by(|| panic!());
-   |               ^^^^^^^ -- takes 0 arguments
-   |               |
+   |               ^^^^^^^ --
+   |               |       |
+   |               |       takes 0 arguments
+   |               |       help: change the closure to take and ignore the arguments: `|_, _|`
```

Comment:
@estebank the position of this help message is a bit weird since I `span_suggestion_with_applicability` the same way  as the tuple-args do. what I want:
```
    |
 help: change the closure to take multiple arguments instead of a single tuple
    |
 LL |     [1, 2, 3].sort_by(|tuple, tuple2| panic!());
| ^^^^^^^^^^^^^^^
```
How to fix this?

Reply:
Suggestions that have [fewer than 10 words (among other things)](https://github.com/rust-lang/rust/blob/487e961c6ae072d969e148def0b0856c2367ae00/src/librustc_errors/emitter.rs#L49-L75) are emitted as above, inline. Making the message longer is one of the ways of ensuring the separate suggestion line.